Package.swift

```diff
@@ -0,0 +1,47 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "TSMessages",
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "TSMessages",
+            targets: ["TSMessages"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
+        .target(
+            name: "TSMessages",
+            dependencies: [],
+            path: "TSMessages",
+            exclude: ["Info.plist"],
+            resources: [
+                .copy("Assets/NotificationBackgroundSuccess@2x.png"),
```

Comment:
Don't you just need to provide the "Assets/" path and it will copy everything under there instead of doing each image specifically?

Reply:
yeah, I tried it, then tried "Assets/*.png" but I think it didn't like the @2x in some of the names